src/networking.c

```diff
@@ -3099,6 +3099,7 @@ sds catClientInfoString(sds s, client *client) {
     if (client->flags & CLIENT_NO_EVICT) *p++ = 'e';
     if (client->flags & CLIENT_NO_TOUCH) *p++ = 'T';
     if (client->flags & CLIENT_REPL_RDB_CHANNEL) *p++ = 'C';
+    if (client->flags & CLIENT_INTERNAL) *p++ = 'I';
```

Comment:
@raz-mon the size of `flags` is 17, now `p` might be out of this range.
This one was discovered by coverity.

Reply:
@sundb This was known in development time, @oranagra preferred we don't update the size of the flags, as some of them do not coexist.